src/app/components/header/header.component.scss

```diff
@@ -0,0 +1,76 @@
+@import '../../../styles/variables';
+
+:host {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0.5rem 1rem;
+
+  a {
+    text-decoration: none;
+  }
+
+  .nav-list {
+    display: inherit;
+    flex-direction: row;
+    justify-content: space-evenly;
+  }
+
+  .nav-item {
+    position: relative;
+    padding: 0 2rem;
+    color: black;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 24px;
+
+    &:after {
```

Comment:
I don't think we ever really agreed upon or designed hover and active styling. Let's pull it out so that it doesnt hold up the rest of the PR while we discuss.